Package.swift

```diff
@@ -0,0 +1,57 @@
+// swift-tools-version:5.3
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "FairmaticSDK",
+    platforms: [.iOS(.v13)],
```

Comment:
do we need anything special for iPadOS or is it enough?

Reply:
iOS is enough, suffices both 👍🏻